oracle/price-feeder/oracle/oracle.go

```diff
@@ -189,7 +189,7 @@ func (o *Oracle) GetPrices() sdk.DecCoins {
 // to determine prices. If candles are not available, uses the most recent prices
 // with VWAP. Warns the user of any missing prices, and filters out any faulty
 // providers which do not report prices or candles within 2𝜎 of the others.
-func (o *Oracle) SetPrices(ctx context.Context) error {
+func (o *Oracle) SetPrices(ctx context.Context, whitelist oracletypes.DenomList) error {
```

Comment:
maybe it would be better to store this `whitelist` in the Oracle struct? and then when we re-fetch params we can just update that?

Reply:
I now have it reading from the paramCache which is already on the oracle